WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -221,7 +221,11 @@ + (NSArray *)routes
 {
   FBElementCache *elementCache = request.session.elementCache;
   XCUIElement *element = [elementCache elementForUUID:request.parameters[@"uuid"]];
-
+  
+  //if element is not specified then use root element i.e. application
+  if(element == nil) {
```

Comment:
Thanks. I was trying to do something like what you already have  /tap/uuid where you ignore uuid. I didn't notice that scroll is like /uuid/scroll and not scroll/uuid. I am doing this support scroll call from appium which doesnt not take element. So i was just going to call /0/scroll.
Anyways, I can change this to have separate custom endpoint just /scroll. Is that okay with you guys?